ide/STM32CUBE/README.md

```diff
@@ -0,0 +1,22 @@
+# wolfSSH for STM32 Cube IDE 
+
+The wolfSSH Cube Pack can be found [here](https://www.wolfssl.com/files/ide/I-CUBE-wolfSSH.pack)
```

Comment:
What STM32 hardware platform(s) have you tested with?

Reply:
Just the STM32F217 so far